package.json

```diff
@@ -38,10 +42,12 @@
     "readme-to-test": "justmoon/readme-to-test#feature/st-improvements",
     "sinon": "^1.17.4",
     "validate-commit-msg": "^2.1.0",
-    "watchify": "^3.7.0"
+    "watchify": "^3.7.0",
+    "webpack": "^1.13.3"
   },
   "scripts": {
     "lint": "eslint .",
+    "prepublish": "webpack --config webpack.config.js",
```

Comment:
Make this a separate script called `build:web` and then call it from `prepublish`.